src/test/java/ch/wisv/areafiftylan/TicketRestIntegrationTest.java

```diff
@@ -429,4 +430,48 @@ private User createTeamReturnSingleMember() {
 
         return teamMate;
     }
+
+    @Test
+    public void testGetAllTicketsForTransport_Anon() {
+        //@formatter:off
+        when()
+            .get("/tickets/transport")
+        .then()
+            .statusCode(HttpStatus.SC_FORBIDDEN);
+        //@formatter:on
+    }
+
+    @Test
+    public void testGetAllTicketsForTransport_User() {
+        SessionData login = login("user", userCleartextPassword);
+
+
+        //@formatter:off
+        given()
+            .filter(sessionFilter)
+            .header(login.getCsrfHeader())
+        .when()
+            .get("/tickets/transport")
+        .then()
+            .statusCode(HttpStatus.SC_FORBIDDEN);
+        //@formatter:on
+    }
+
+    @Test
+    public void testGetAllTicketsForTransport_Admin() {
```

Comment:
The `ticketRepository` is initialized with a ticket without transport. This can be seen in the `@before` method. The `ticketRepository` contains 2 tickets when executing this test. That should be enough, or am I missing something?